wweval/R/ms_fig4.R

```diff
@@ -262,71 +333,80 @@ make_fig4_rel_crps_overall <- function(scores,
                                          "nowcast",
                                          "1 wk", "4 wks",
                                          "overall"
-                                       )) {
-  scores_by_horizon <- scores |>
-    data.table::as.data.table() |>
-    scoringutils::summarise_scores(by = c(
-      "forecast_date", "location",
-      "model", "horizon"
-    ))
-
-  scores_overall <- scores |>
-    data.table::as.data.table() |>
-    scoringutils::summarise_scores(by = c(
-      "forecast_date", "location",
-      "model"
-    )) |>
-    dplyr::mutate(horizon = "overall")
+                                       ),
+                                       summarize_across_horizon = FALSE) {
+  if (isTRUE(summarize_across_horizon)) {
+    scores_by_horizon <- scores |>
+      data.table::as.data.table() |>
+      scoringutils::summarise_scores(by = c(
+        "forecast_date", "location",
+        "model", "horizon"
+      ))
+
+    scores_overall <- scores |>
+      data.table::as.data.table() |>
+      scoringutils::summarise_scores(by = c(
+        "forecast_date", "location",
+        "model"
+      )) |>
+      dplyr::mutate(horizon = "overall")
+  } else {
+    scores_by_horizon <- scores
+    scores_overall <- scores |>
+      dplyr::mutate(
+        horizon = "overall"
+      )
+  }
```

Comment:
When / how do you use this switch?

Reply:
I think we're unlikely to go back and if we do we may well want a separate panel creation function (or want two functions that call the same internal helper after summarizing things differently). So I think you can delete the switch.